src/resolvers/Mutation/acceptMembershipRequest.ts

```diff
@@ -94,19 +100,18 @@ export const acceptMembershipRequest: MutationResolvers["acceptMembershipRequest
         $push: {
           members: user._id,
         },
-
-        $set: {
-          membershipRequests: organization.membershipRequests.filter(
-            (request) => request.toString() !== membershipRequest._id.toString()
-          ),
+        $pull: {
+          membershipRequests: membershipRequest._id,
         },
       }
     );
 
-    /*
-   Add organization._id to joinedOrganizations list and remove 
-   membershipRequest._id from membershipRequests list on user's document.
-   */
+    /* This code is updating the `User` document in the database by adding the `organization._id` to
+    the `joinedOrganizations` array and removing the `membershipRequest._id` from the
```

Comment:
These comments are useless, any person using mongoose should have an idea of what's being done. Even if you're doing it these comments can still be trimmed to a considerable amount.

Reply:
Okay sir